pkg/controllers/hub/trafficmanagerbackend/controller_integration_test.go

```diff
@@ -26,8 +26,9 @@ import (
 )
 
 const (
-	timeout  = time.Second * 10
-	interval = time.Millisecond * 250
+	timeout     = time.Second * 10
```

Comment:
can we define a common base line 10 second timeout for all tests instead of each have its own?

Reply:
do you mean creating a common variable somewhere and use that for all the controllers?

it's doable, but the timeout may not be necessarily the same and depends on the tests & implementations.

Reply:
Are you suggesting to create a common timeout for all e2e tests?

These timeout may not be applied to integration tests, cause in the integration test, we use the fake arm call.

Reply:
yes, all e2e follow the same pattern

Reply:
sure, we can do the refactor in a separate PR. This comment was put on the IT which confused me :)